external/header.php

```diff
@@ -43,15 +43,18 @@
  *  Adds more information about function calls (this information is not currently used by XHGui)
  */
 
-/* Tideways support
- * The tideways extension is a fork of xhprof. See https://github.com/tideways/php-profiler-extension
+/* Tideways XHProf support
+ * The tideways_xhprof extension is a fork of xhprof. See https://github.com/tideways/php-profiler-extension
  *
  * It works on PHP 5.5+ and PHP 7 and improves on the ancient timing algorithms used by XHProf using
  * more modern Linux APIs to collect high performance timing data.
  *
- * The TIDEWAYS_* constants are similar to the ones by XHProf, however you need to disable timeline
+ * v4 (tideways): The TIDEWAYS_* constants are similar to the ones by XHProf, however you need to disable timeline
  * mode when using XHGui, because it only supports callgraphs and we can save the overhead. Use
  * TIDEWAYS_FLAGS_NO_SPANS to disable timeline mode.
+ *
+ * v5 (tideways_xhprof): The TIDEWAYS_XHPROF_* constants are similar to the ones by XHProf, however you cannot use
+ * additional TIDEWAYS_XHPROF_FLAGS_MEMORY_* flags since XHGui does not support the extra data that they produce.
```

Comment:
is this comment correct?

> however you cannot use additional `TIDEWAYS_XHPROF_FLAGS_MEMORY_*` flags since XHGui does not support the extra data that they produce.

because if it is, then why you still enable `TIDEWAYS_XHPROF_FLAGS_MEMORY` below?